recirq/otoc/loschmidt/tilted_square_lattice/analysis-walkthrough.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>Should this header be one level lower?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/ReCirq/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>no, I don't think so. The title is &lt;h1&gt;, most things are &lt;h2&gt; and only minor diversions are &lt;h3&gt;</p>